test/command_line/helper/command_runner.rb

```diff
@@ -103,8 +103,9 @@ def grndb(command, *arguments)
   end
 
   def find_program(name, options={})
+    name = name + RbConfig::CONFIG["EXEEXT"]
     ENV["PATH"].split(File::PATH_SEPARATOR).each do |path|
-      program_path = File.join(path, name)
+      program_path = [path, name].join(File::ALT_SEPARATOR || File::SEPARATOR)
```

Comment:
Could you use same logic for consistency?

```suggestion
      separator = File::ALT_SEPARATOR || File::SEPARATOR
      program_path = [path, name].join(separator)
      libs_lt_program_path = [path, ".libs", "lt-#{name}"].join(separator)
      libs_program_path = [path, ".libs", name].join(separator)
```

Reply:
It is reasonable.
I've fixed it.